pystac/catalog.py

```diff
@@ -167,6 +171,9 @@ class Catalog(STACObject):
     a canonical format.
     """
 
+    _layout_strategy: HrefLayoutStrategy = BestPracticesLayoutStrategy()
```

Comment:
So the idea is that pystac will be able to override this in the `Client` class?

Reply:
Yes, correct. For example, the `pystac_client.Client` could set the fallback strategy to `APILayoutStrategy`